Description: Fix build issue on kFreeBSD
 This patch fixes a build issue on kFreeBSD. It is caused
 by an undefined variable which was forgotten during a code
 rewrite.
Author: Manuel Prinz <manuel@debian.org>
Forwarded: https://svn.open-mpi.org/trac/ompi/ticket/2493
Last-Update: 2010-07-30

--- a/opal/mca/memory/ptmalloc2/opal_ptmalloc2_munmap.c
+++ b/opal/mca/memory/ptmalloc2/opal_ptmalloc2_munmap.c
@@ -92,7 +92,6 @@
         
             tmp.munmap_p = dlsym(RTLD_NEXT, "munmap");
             realmunmap = tmp.munmap_fp;
-            ++count;
         }
 
         return realmunmap(start, length);
